### PR TITLE
Allow user to minimize panes by clicking menu button #1154

### LINKED ISSUE #1154: Allow user to minimize panes by clicking menu button
Currently, clicking the Resize/Shapes/Colors Lab menu button when the respective pane is open doesn't do anything, but for Positions Lab, it closes the respective pane.

The program would feel more responsive and consistent if the user could use the menu buttons to close the respective pane.

### PR DESCRIPTION
**Pull request by jamos-tay**
Fixes #1154.

Clicking the menu button now closes the respective pane if it is open.

### CODE REVIEW

File: PowerPointLabs/PowerPointLabs/Ribbon1.cs [hunk 1]
```diff
@@ -1949,7 +1949,7 @@ public void AddShapeButtonClick(Office.IRibbonControl control)
             customShape.AddCustomShape(shapeName, shapeFullName, true);
         }
 
-        private static CustomShapePane InitCustomShapePane()
+        private static CustomShapePane InitCustomShapePane(bool toggleVisible = false)
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/Ribbon1.cs]: is this param necessary?

Comment (jamos-tay) [inline PowerPointLabs/PowerPointLabs/Ribbon1.cs]: Yeah, this function is also being called by `AddShapeButtonClick` which is when the user tries to add a shape to shapes lab. We don't want the pane to closed when the user adds shapes, so I made a boolean to check.

Comment (kai33) [inline PowerPointLabs/PowerPointLabs/Ribbon1.cs]: This method is mixing the initialization of shapes lab and the UI visibility control. Can u refactor it ?

Comment (jamos-tay) [inline PowerPointLabs/PowerPointLabs/Ribbon1.cs]: Okay, I added two functions for setting visibility.
Tests are passing

File: PowerPointLabs/PowerPointLabs/Ribbon1.cs [hunk 2]
```diff
@@ -1920,6 +1920,7 @@ public void RemoveAllNotesClick(Office.IRibbonControl control)
         public void CustomShapeButtonClick(Office.IRibbonControl control)
         {
             InitCustomShapePane();
+            TogglePaneVisibility();
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/Ribbon1.cs]: LGTM.
Can you also refactor these codes into handler under action framework?

Comment (jamos-tay) [inline PowerPointLabs/PowerPointLabs/Ribbon1.cs]: Okay, changed ShapesLab and AddCustomShape to action framework.
Tests passing on 2013

File: PowerPointLabs/PowerPointLabs/ActionFramework/Action/ShapesLabActionHandler.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            var customShapePane = GetShapesLabPane();
+
+            if (customShapePane == null)
+            {
+                return null;
+            }
+
+            var customShape = customShapePane.Control as CustomShapePane;
+
+            Trace.TraceInformation(
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/ActionFramework/Action/ShapesLabActionHandler.cs]: use Logger

**Pull request merged**